tests/acceptance/cli/test_alice.py

```diff
@@ -114,28 +113,28 @@ def test_initialize_alice_with_custom_configuration_root(custom_filepath, click_
     assert 'IPv4' not in result.output
 
     # Files and Directories
-    assert os.path.isdir(custom_filepath), 'Configuration file does not exist'
-    assert os.path.isdir(os.path.join(custom_filepath, 'keystore')), 'KEYSTORE does not exist'
-    assert os.path.isdir(os.path.join(custom_filepath, 'known_nodes')), 'known_nodes directory does not exist'
+    assert custom_filepath.is_dir(), 'Configuration file does not exist'
+    assert (custom_filepath / 'keystore').is_dir(), 'Keystore does not exist'
+    assert (custom_filepath / 'known_nodes').is_dir(), 'known_nodes directory does not exist'
 
-    custom_config_filepath = os.path.join(custom_filepath, AliceConfiguration.generate_filename())
-    assert os.path.isfile(custom_config_filepath), 'Configuration file does not exist'
+    custom_config_filepath = custom_filepath / AliceConfiguration.generate_filename()
+    assert custom_config_filepath.is_file(), 'Configuration file does not exist'
 
     # Auth
     assert COLLECT_NUCYPHER_PASSWORD in result.output, 'WARNING: User was not prompted for password'
     assert 'Repeat for confirmation:' in result.output, 'User was not prompted to confirm password'
 
 
 def test_alice_control_starts_with_preexisting_configuration(click_runner, custom_filepath):
-    custom_config_filepath = os.path.join(custom_filepath, AliceConfiguration.generate_filename())
+    custom_config_filepath = custom_filepath / AliceConfiguration.generate_filename()
     run_args = ('alice', 'run', '--dry-run', '--lonely', '--config-file', custom_config_filepath)
```

Comment:
Am I missing something or do we need to update the value in args or else it would be a Path object...? Here and in other spots.

```suggestion
    run_args = ('alice', 'run', '--dry-run', '--lonely', '--config-file', str(custom_config_filepath.absolute()))
```

Reply:
`Path` objects passed to CLI runner seem to be working. But I think it would be better to maintain the proposed convention of  passing `str` containing absolute paths instead.